Documentation/Contributors/CodingGuide/README.md

```diff
@@ -544,6 +544,21 @@ var p = new Cartesian3(1.0, 2.0, 3.0);
 p.x = 'Cesium'; // Changes x to a string, slows down property access
 ```
 
+* In a constructor function, consider properties as write once; do not write to them multiple times or read them. Create a local if they need to be ready. For example:
```

Comment:
Just some nick pick on wording:

"do not write to them multiple times or read them" -> "do not write to them or read them multiple times"

"Create a local" -> "Create a local variable".